tests/test_loaders.py

```diff
@@ -67,15 +67,15 @@ def loader_module():
     return loaders.PluginLoader(
         content_type=constants.ContentType.MODULE,
         rel_path='plugins/modules/my_sample_module.py',
-        root='/tmp/tmpiskt5e2n')
+        root='/tmp/tmpiskt5e2n/ansible_collections/my_ns/my_collection')
```

Comment:
The tmp path here should probably be from one of the pytest tmpfile fixtures.

A static predictable /tmp path can be dangerous (ie, something/someone else could create a  '/tmp/tmpiskt5e2n' that is link elsewhere [/etc/passwd' or '/root/.ssh' are classic examples] that would get clobbered when a dev or ci runs the unit tests).

Granted, that is not nearly as dangerous now as it used to be thanks to per user /tmp, containers, etc. But should probably be fixed.

Reply:
This path is only used for string manipulation, but I renamed it so it is not /tmp and more clearly a placeholder